browse/factory.py

```diff
@@ -6,7 +6,7 @@
 from arxiv.base.urls import canonical_url, clickthrough_url, urlizer
 from arxiv.base.config import BASE_SERVER
 from arxiv.base import Base
-from arxiv.users.auth import Auth
+# from arxiv.users.auth import Auth
```

Comment:
This commented out due to the "Restore support of login status in Browse" issue.
https://arxiv-org.atlassian.net/browse/ARXIVNG-4354